db/migrations/20181227100332_add_fxa_columns.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table("subscribers", table => {
+    table.string("fxa_refresh_token");
+    table.json("fxa_profile_json");

```

Comment:
You should be able to use `table.jsonb` here, as https://github.com/tgriesser/knex/issues/991 was fixed in https://github.com/tgriesser/knex/pull/1044

Reply:
Updated to `jsonb`

Reply:
While this should work in 9.4, https://github.com/crowdAI/crowdai/issues/68#issuecomment-285289202 needed to switch to Postgres 9.6 (and ubuntu trusty) to get jsonb working on travisci.